models/method.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/iancoleman/strcase"
+	"github.com/lightninglabs/lightning-api-ng/defs"
+	"github.com/lightninglabs/lightning-api-ng/markdown"
+)
+
+type Method struct {
+	Service *Service
+
+	Name               string
+	Description        string
+	Source             string
+	CommandLine        string
+	CommandLineHelp    string
+	RequestType        string
+	RequestFullType    string
+	RequestTypeSource  string
+	RequestStreaming   bool
+	ResponseType       string
+	ResponseFullType   string
+	ResponseTypeSource string
+	ResponseStreaming  bool
+}
+
+// NewMethod creates a new method from a method definition
+func NewMethod(methodDef *defs.ServiceMethod, service *Service) *Method {
+	m := &Method{}
```

Comment:
Same here, I prefer the direct struct initialization, especially as it could also be returned directly:
```
return &Method{
    Service: service,
....
}
```